logger.js

```diff
@@ -1,13 +1,29 @@
 const bunyan = require('bunyan');
+const fluentL = require('fluent-logger');
 
-module.exports = function createLogger({ name }) {
+module.exports = function createLogger({ name, output, host, port }) {
   if (!name) {
     throw new Error('Please provide a name');
   }
 
+  let stream = process.stdout;
+
+  if (output === 'td-agent-forward') {
+    const sender = fluentL.createFluentSender('', {
+      host,
+      port,
+      timeout: 3.0,
+      levelTag: false,
+      reconnectInterval: 600000,
+    });
+    stream = sender.toStream('application.logs');
+  } else {
```

Comment:
We likely could remove this else blog if we're setting a default stream above in the `let` declaration.